docs/src/pages/_app.page.tsx

```diff
@@ -38,6 +39,7 @@ if (typeof window === 'undefined') {
 }
 
 function MyApp({ Component, pageProps }) {
+  const stage = getStage();
```

Comment:
Let's have `IS_PROD_STAGE`, `IS_DEV_STAGE` constants that call `getStage === 'prod'` once, then we don't have to do string checks on 75.